pkg/apis/v1alpha2/aksnodeclass.go

```diff
@@ -53,8 +53,8 @@ type AKSNodeClassSpec struct {
 	// +optional
 	Kubelet *KubeletConfiguration `json:"kubelet,omitempty" hash:"ignore"`
 	// MaxPods is an override for the maximum number of pods that can run on a worker node instance.
-	// +kubebuilder:validation:Minimum:=0
-	// +kubebuilder:default=250
+	// +kubebuilder:validation:Minimum:=10
```

Comment:
Please add a comment on reasoning (I presume this matches AKS-enforced minimum?)

(This is also a breaking change, but an acceptable one I think)

Reply:
Added much better comments on 

A: Min 
B: Max 
C: Default